apps/web/src/components/tx/ConfirmTxDetails/TxDetails.tsx

```diff
@@ -157,6 +155,43 @@ export const TxDetails = ({ safeTx, txData, showHashes }: TxDetailsProps) => {
                 </TxDetailsRow>
 
                 <TxDetailsRow label="Nonce">{safeTx.data.nonce}</TxDetailsRow>
+
+                <Button onClick={() => setShowHashes(!showHashes)} sx={{ all: 'unset' }}>
+                  <Typography
+                    variant="body2"
+                    fontWeight={700}
+                    display="inline-flex"
+                    alignItems="center"
+                    color="primary.light"
+                    sx={{ cursor: 'pointer' }}
+                  >
+                    Transaction hashes <ExpandMoreIcon sx={{ transform: !showHashes ? 'rotate(180deg)' : undefined }} />
```

Comment:
```suggestion
                    Transaction hashes <ExpandMoreIcon sx={showHashes ? { transform: 'rotate(180deg)' } : undefined }} />
```

In the collapsed state, the arrow should face down.

Reply:
Fixed in 8369eb9d3ff46dae34824fe69e7dd15b4e828fb2.